### ISSUE TITLE
[Bug] Expiration Date edit box do not have an a11y label for year and month

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/fenix#19820]
## Steps to reproduce
1. Reach the `Add card` section with TalkBack ON.
2. Tap on the `Expiration Date` drop-down lists, month, and year.
### Expected behavior
Input edit boxes should have a label as a hint to what is expected to be inputted by the user.
### Actual behavior
Only drop-down list is read when tapping on the `Expiration Date` month and year.
### Device information

* Device vendor: Google Pixel 4 XL (11).
* Firefox for Android version: Nightly 6/4.

**Comment by abodea** [comment]
Verified as fixed on the latest Nightly 6/9 with Google Pixel 4 XL (11).
Please note that the Expiration date now is read correctly.
